Task :react-native-user-agent:verifyReleaseResources FAILED

> Task :react-native-user-agent:verifyReleaseResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-user-agent:verifyReleaseResources'.
> 1 exception was raised by workers:
  com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking faile
  /Users/abc/Documents/iGamio/node_modules/react-native-user-agent/android/build/intermediates/res/merged/release/values-v28/values-v28.xml:7: error: resource android:attr/dialogCornerRadius not found.
  /Users/abc/Documents/iGamio/node_modules/react-native-user-agent/android/build/intermediates/res/merged/release/values-v28/values-v28.xml:11: error: resource android:attr/dialogCornerRadius not found.
  /Users/abc/Documents/iGamio/node_modules/react-native-user-agent/android/build/intermediates/res/merged/release/values/values.xml:2743: error: resource android:attr/fontVariationSettings not found.
  /Users/abc/Documents/iGamio/node_modules/react-native-user-agent/android/build/intermediates/res/merged/release/values/values.xml:2744: error: resource android:attr/ttcIndex not found.
  error: failed linking references.
  

I am using 
    "react": "16.9.0",
    "react-dom": "^16.12.0",
    "react-native": "0.61.5",
    "react-native-user-agent": "^2.1.0",


I am facing this issue. Please help me in resolve this. Thank you.

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -1,3 +1,6 @@
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
 
 buildscript {
     repositories {
@@ -12,15 +15,16 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 27
-    buildToolsVersion "27.0.3"
+    compileSdkVersion safeExtGet('compileSdkVersion', 27)
+    buildToolsVersion safeExtGet('buildToolsVersion', '27.0.3')
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 22
+        minSdkVersion safeExtGet('minSdkVersion', 16)
+        targetSdkVersion safeExtGet('targetSdkVersion', 22)
         versionCode 1
         versionName "1.0"
     }
+
     lintOptions {
         abortOnError false
     }
@@ -33,4 +37,3 @@ repositories {
 dependencies {
     compile 'com.facebook.react:react-native:+'
 }
-  

